python/ray/autoscaler/local/example-full.yaml

```diff
@@ -49,11 +39,24 @@ auth:
     # Optional if an ssh private key is necessary to ssh to the cluster.
     # ssh_private_key: ~/.ssh/id_rsa
 
-# Leave this empty.
-head_node: {}
+# The minimum number of worker nodes to add to the Ray cluster in addition to the head
+# node. This number should be >= 0.
+min_workers: MIN_NUMBER_OF_NODES_TO_KEEP_IN_THE_CLUSTER
```

Comment:
can you set it to the default and say what is the default as it was previously?

Reply:
Will revert to [current](https://github.com/ray-project/ray/blob/59bf3a7b22b450f9c89caa16c6b46a0ebc6c4918/python/ray/autoscaler/local/example-full.yaml#L6-L14) state which is to set both to 0 and say what the typical desired behavior is.

Reply:
Done.